<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegacyAdvancedSettingsWidget</class>
 <widget class="QWidget" name="LegacyAdvancedSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="nx::vms::client::desktop::LegacyCameraSettingsStreamsPanel" name="streamsPanel" native="true"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="noSettingsPage">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="noSettingsLabel">
         <property name="text">
          <string notr="true">&lt;This camera has no advanced settings&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="manualPage">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <item>
        <widget class="nx::vms::client::desktop::CameraAdvancedParamsWidget" name="cameraAdvancedParamsWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="webPage">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWebView" name="webView" native="true">
         <property name="url" stdset="0">
          <url>
           <string/>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::CameraAdvancedParamsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/camera_advanced_params_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::LegacyCameraSettingsStreamsPanel</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/resource_properties/camera/legacy/legacy_camera_settings_streams_panel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
